fluentui_core/src/main/java/com/microsoft/fluentui/util/AccessibilityUtils.kt

```diff
@@ -118,34 +118,18 @@ fun Modifier.clickableWithTooltip(
     val currentOnLongClick by rememberUpdatedState(onLongClick)
     val currentTooltipEnabled by rememberUpdatedState(tooltipEnabled)
 
-    val indicationModifier = Modifier.indication(
+    val clickableModifier = Modifier.combinedClickable(
```

Comment:
what does this combinedClickable do? and i see you have added experimentalAPI, would it be safe to use that?